Sources/EpoxyBars/BarInstaller/BarStackView.swift

```diff
@@ -86,6 +88,55 @@ public class BarStackView: UIStackView {
     return nil
   }
 
+  public override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
+    super.touchesBegan(touches, with: event)
+    guard let location = touches.first?.location(in: self) else { return }
+
+    for wrapper in wrappers {
+      guard let converted = wrapper.view?.convert(location, from: self) else { continue }
+      if wrapper.view?.point(inside: converted, with: event) == true {
+        selectedWrapper = wrapper
+        break
+      }
+    }
```

Comment:
I think we should only set `selectedWrapper` if `wrapper` has a `_model` that has a non-`nil` `didSelect` closure—could we maybe add a property on `BarWrapper` like `var canHighlight: Bool` that we check first here?
```suggestion
    for wrapper in wrappers where wrapper.canHighlight {
      guard let converted = wrapper.view?.convert(location, from: self) else { continue }
      if wrapper.view?.point(inside: converted, with: event) == true {
        selectedWrapper = wrapper
        break
      }
    }
```